scripts/test/terra_validation.py

```diff
@@ -84,21 +83,27 @@ def validate_terra_json(wdl, terra_json, womtool_jar):
     if valid:
         logging.info(f"PASS: {terra_json} is a valid Terra input JSON for {wdl_name}")
 
-    return int(valid)  # return 1 if valid, 0 if not valid
+    return valid
 
 
 def validate_all_terra_jsons(base_dir, womtool_jar, expected_num_inputs):
     successes = 0
+    failures = 0
     for wdl, json_file in get_wdl_json_pairs(os.path.join(base_dir, WDLS_PATH),
                                              [os.path.join(base_dir, x) for x in TERRA_INPUTS_PATHS],
                                              expected_num_inputs):
-        successes += validate_terra_json(wdl, json_file, womtool_jar)
+        valid = validate_terra_json(wdl, json_file, womtool_jar)
+        if valid:
+            successes += 1
+        else:
+            failures += 1
 
     print("\n")
-    if successes != expected_num_inputs:
-        raise RuntimeError(f"{successes} of {expected_num_inputs} Terra input JSONs passed validation.")
-    else:
-        print(f"{successes} of {expected_num_inputs} Terra input JSONs passed validation.")
+    if failures > 0:
+        raise RuntimeError(f"Some Terra input JSONs failed validation!\nPass: {successes}. Fail: {failures}.\nTotal validated: {successes + failures}. Expected number of JSONs: {expected_num_inputs}.")
+    if successes > expected_num_inputs:
+        logging.warning(f"Found more Terra input JSONs than expected! Validated: {successes}. Expected: {expected_num_inputs}.")
+    print(f"Success! All Terra input JSONs passed validation.\nPass: {successes}. Fail: {failures}.\nTotal: {successes + failures} out of {expected_num_inputs} passed.")
```

Comment:
What if there are no errors, but successes < expected_num_inputs, ie in the case where `-n` is simply too high?

Reply:
I tested that - it errors out on line 44 with the check for `if num_input_jsons < expected_num_jsons`. I added to that error message saying to increase `-n`.